ansible/group_vars/k8strain-hosts.yml

```diff
@@ -0,0 +1,245 @@
+# Kubernetes configuration dirs and system namespace.
+# Those are where all the additional config stuff goes
+# the kubernetes normally puts in /srv/kubernets.
```

Comment:
typo ``kubernets``

Reply:
This typo is present in the original sample copied file https://github.com/kubernetes-incubator/kubespray/blob/d8d5474dcc0b492946b4fd29e1c0a51ad08c8c20/inventory/sample/group_vars/k8s-cluster.yml, I've minimised changes to only the required config